images/kubeflow-katib-file-metricscollector/tests/main.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_string" "ref" { input = var.digest }
+
+data "oci_exec_test" "runs" {
+  digest = var.digest
+  script = "docker run --rm $IMAGE_NAME --help"
```

Comment:
This is reasonable as a basic test, but I think we need to make sure we test this more thoroughly with some kind of real usage before merging this.